src/util/write-payload.ts

```diff
@@ -33,9 +33,9 @@ export class WritePayload<T extends JSORMBase> {
     const attrs: ModelRecord<T> = {}
 
     this._eachAttribute((key, value) => {
-      const snakeKey = underscore(key)
+      const snakeKey: keyof ModelRecord<T> = underscore(key) as any
```

Comment:
would it be better to put `string` instead of `as any` here?

Reply:
need to convert to `any` to assign it back to the `keyof ModelRecord<T>`. It's a string before the `any` cast, and typescript doesn't like it.